src/arrays/ArrayUtil.java

```diff
@@ -26,8 +39,8 @@ public static void create2dArray(int x, int y) {
      * @return new array.
      */
     public static int[] addFirst(int[] array, int valueToAdd) {
-
-        return null;
+        array[0] = valueToAdd;
```

Comment:
Hayk jan you should add the value to the first position, and move others by one index right.
Here you just override first element